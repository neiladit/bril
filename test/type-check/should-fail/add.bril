main{
    a: int  = const 1;
    b: bool = const true;
    c: int = add a b;
    print c;
}
