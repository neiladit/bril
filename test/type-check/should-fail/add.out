line 3: args b is not expected type int
arithmetic op error!
